github-usercard/src/App.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from "react";
+import axios from "axios";
+import Followers from "./Components/followers";
+import styled from "styled-components";
+
+const Div = styled.div`
+  width: 1200px
+  margin: 0 auto;
+  text-align: center;
+`;
+const Div2 = styled.div`
+  display: flex;
+  text-align: left;
+`;
+const Div3 = styled.div`
+  text-align: left;
+  margin-left: 2rem;
+`;
+const Div4 = styled.div`
+  margin: 2rem 0;
+`;
+
+const Img = styled.img`
+  max-width: 350px;
+  max-height: 350px;
+  border-radius: 5px;
+  margin: auto 0;
+`;
+
+export default class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      follow: []
+    };
+  }
+
+  getGitUser = () => {
+    axios
+      .get("https://api.github.com/users/JacobWilliams90")
+      .then(res => {
+        this.setState({
+          data: res.data
+        });
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  };
+
+  gitGitFollowers = () => {
+    axios
+      .get("https://api.github.com/users/JacobWilliams90/followers")
+      .then(res => {
+        this.setState({
+          follow: res.data
+        });
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  };
+
+  componentDidMount() {
+    this.getGitUser();
+    this.gitGitFollowers();
+  }
```

Comment:
because you're always getting both, I might consider putting both axios in 1 function.  Also, the downside of doing that is that you're setState twice which means that your app is rerendering twice.  A way to get around that is to set the res.data to a variable and then after the axios call setState once with both variables.  Or you can use a Promise.all()

```getData() {
    let urlArray = [
      `https://api.github.com/users/${this.state.githubHandle}`,
      `https://api.github.com/users/${this.state.githubHandle}/followers`
    ];
    // Promise.all() takes in an array of promises and outputs an array of the returned promises
    Promise.all(
      urlArray.map(url =>
        // Example of using Javascript fetch api <- from Vlad
        // fetch(url)
        // .then(response => response.json()) // response is another promise that can be resolved with .json()
        //   .catch(err => console.error("Request failed", err))

        // example of using axios api
        axios
          .get(url)
          .then(res => res.data)
          .catch(err => console.log("error: ", err))
      )
    ).then(dataArray => {
      console.log(dataArray);
      this.setState(
        {
          userObj: dataArray[0],
          followers: dataArray[1]
        },
        () => console.log("state: ", this.state)
      );
    });
  }```